app/pages/lab/subject-set.cjsx

```diff
@@ -17,9 +17,6 @@ VALID_SUBJECT_EXTENSIONS = ['.jpg', '.jpeg', '.png', '.gif', '.svg']
 INVALID_FILENAME_CHARS = ['/', '\\', ':', ',']
 MAX_FILE_SIZE = 600 * 1024
 
-announceSetChange = ->
-  apiClient.type('subject_sets').emit 'add-or-remove'
```

Comment:
I'm guessing it empties the classification subject queue when you log out, since it's hooked up to auth change.